projects/netgrif-components-core/src/assets/i18n/de.json

```diff
@@ -390,6 +390,7 @@
     },
     "workflow": {
         "delete": "löschen",
+        "download": "heruntergeladen",
```

Comment:
the verb should not be in the past tense. The correct translation is `herunterladen`